nomad/alloc_endpoint.go

```diff
@@ -244,6 +244,8 @@ func (a *Alloc) Stop(args *structs.AllocStopRequest, reply *structs.AllocStopRes
 		JobID:          alloc.Job.ID,
 		JobModifyIndex: alloc.Job.ModifyIndex,
 		Status:         structs.EvalStatusPending,
+		CreateTime:     time.Now().UTC().UnixNano(),
+		ModifyTime:     time.Now().UTC().UnixNano(),
```

Comment:
For all of these that set CreateTime and ModifyTime in the same place, the times will be milliseconds different. That seems wrong? ModifyTime should probably use the same timestamp as CreateTime for eval creation

Reply:
I agree. It would be weird, at the very least.

Reply:
I think that either using the same timestamp or leaving `ModifyTime: 0` would more useful, because it allows us to ask if an eval is unmodified.

Reply:
+1 to creating a variable for time.Now().UTC().UnixNano() above and using it for both create and modify time here and in other spots

Reply:
Fixed.